packages/lightning-lsp-common/src/shared.ts

```diff
@@ -73,25 +73,39 @@ export function detectWorkspaceHelper(root: string): WorkspaceType {
         return WorkspaceType.CORE_PARTIAL;
     }
 
+    if (fs.existsSync(path.join(root, 'lwc.config.js'))) {
```

Comment:
@xyc @mrmicahcooper here it should also have been 'lwc.config.json'

Reply:
Thanks we'll take a look. @muenzpraeger also mentions it

Reply:
It's already fixed in #16 (sorry ;-)